homeassistant/components/point/__init__.py

```diff
@@ -181,8 +181,9 @@ def __init__(self, hass: HomeAssistantType, config_entry: ConfigEntry,
             async with self._hass.data[DATA_CONFIG_ENTRY_LOCK]:
                 if config_entries_key not in self._hass.data[
                         CONFIG_ENTRY_IS_SETUP]:
-                    await self._hass.config_entries.async_forward_entry_setup(
-                        self._config_entry, component)
+                    self._hass.async_create_task(
```

Comment:
We should still await this to ensure that the forwarding is done before we release the lock. Instead we should create a task for the first `client.update`.

https://github.com/home-assistant/home-assistant/blob/8fe95f4babbd5022bed41f3f3b4b08f9cf3486d6/homeassistant/components/point/__init__.py#L91